README.md

```diff
@@ -449,6 +459,30 @@ make makemigrations
 make migrate
 ```
 
+### Translations
+
+Translations are managed using .po files, which are compiled into .mo files for use in the application.
+The .po files are located in the `cms/locale` directory.
+
+If you add new text to the application, you will need to update the .po files to include the new text.
+You can do this by running the following command:
+
+```bash
+make makemessages
```

Comment:
This is currently erroring for me:
```
UnicodeDecodeError: skipped file ThirdPartyNoticeText.txt in ./node_modules/typescript (reason: 'utf-8' codec can't decode byte 0xa0 in position 6038: invalid start byte)
```
The following fixed it:
```sh
❯ poetry run python ./manage.py makemessages --locale cy --ignore "node_modules/*"
```

Reply:
Thanks! Addressed in [a1d18ec](https://github.com/ONSdigital/dis-wagtail/pull/140/commits/a1d18ecce89888773e78193cfba1a5a37af39d6d)